test/integration/actions/action-create-event.spec.ts

```diff
@@ -72,6 +74,42 @@ describe('action-create-event', () => {
 			},
 		};
 
+		// Before enqueing up a create event we need to create a relationship to the random-slug type created above
+		const relationship: RelationshipContractDefinition = {
```

Comment:
Could we instead change the types used in the test so we can use a real relationship we expect to exist?